pkg/util/k8sutil/k8sutil.go

```diff
@@ -134,6 +136,16 @@ func ImageName(repo, version string) string {
 	return fmt.Sprintf("%s:v%v", repo, version)
 }
 
+func imageNameBusybox(policy *api.PodPolicy) string {
+	// set defaults for busybox init container
```

Comment:
move the comment on top of `imageNameBusybox`?
and change the comment to be `imageNameBusybox set defaults for busybox init container`